modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -1668,7 +1697,7 @@ <h4><a id="node">Node</a></h4>
           <td class="default">false</td>
           <td class="range">&nbsp;</td>
           <td>The default value for controls is true, although there are some exceptions.
-          See <a href="#controls">Controls</a> for details.</td>
+          See <a href="#control">Control</a> for details.</td>
```

Comment:
As noted by @Maran23 , this change should be reverted and a new section header for "Controls" should be added.

Reply:
not here - here the link should point to Control.

@Maran23 is right - the heading H2 was originally missing from the Controls section (line 3245)

Reply:
Although... the information is also repeated in the Control class, so I won't object if you want to keep your change to point there.

Reply:
pushed the change, try it now.